docker-compose.yml

```diff
@@ -24,18 +21,45 @@ services:
       CONJUR_ACCOUNT: cucumber
 
   cuke-master:
-    container_name: api-go-conjur-v4
     image: registry2.itci.conjur.net/conjur-appliance-cuke-master:4.9-stable
     security_opt:
       - seccomp:unconfined
     ports:
       - 443
 
-  test:
-    container_name: api-go-test
-    build: .
+  test-1.14:
```

Comment:
The docker-compose definition for `test-1.14` looks almost identical to the definition for `test-1.15` service.
Also, I think the `test-1.14` service is run mutually exclusive from the `test-1.15` service.
So I think you can use docker-compose override, i.e., create a base `test` service in docker-compose
that has default `1.15` values, and create a separate `docker-compose.override-1.14.yml` that simply overrides a few of the values in the base definition.

Just a thought. Not a showstopper.

Reply:
I think it's a small enough addition that we can keep them in the same file, if we were gonna do more than two go versions I think I'd definitely want to use the other method you suggest here